### PR TITLE
qmmp: 1.4.4 -> 2.1.2

### PR DESCRIPTION
**Pull request by zendo**
###### Description of changes

https://qmmp.ylsoftware.com/index.php

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/qmmp/default.nix [hunk 1]
```diff
@@ -28,30 +26,32 @@
...
-      libcdio cdparanoia libcddb faad2 ffmpeg wildmidi
+      libcdio cdparanoia libcddb faad2 ffmpeg wildmidi libbs2b libgme
       # output plugins
-      alsa-lib libpulseaudio
+      alsa-lib libpulseaudio pipewire
       # effect plugins
       libsamplerate
     ];
 
+  dontWrapQtApps = true;
```
Comment (bjornfor) [inline pkgs/applications/audio/qmmp/default.nix:53]: Can you say why you add `dontWrapQtApps = true` in the commit message or comment above this line?

Comment (zendo) [reply pkgs/applications/audio/qmmp/default.nix:53]: The previous version also don't have `wrapQtAppsHook`, it looks fine.

**Comment by bjornfor** [comment]
Result of `nixpkgs-review pr 200723` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>qmmp</li>
  </ul>
</details>

**Comment by bjornfor** [comment]
> Result of `nixpkgs-review pr 200723` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
> 1 package failed to build:
> 
>     * qmmp

error: Function called without required argument "libgme" at /home/bf/.cache/nixpkgs-review/pr-200723-2/nixpkgs/pkgs/applications/audio/qmmp/default.nix:7, did you mean "libgee", "libime" or "lib3mf"?
(use '--show-trace' to show detailed location information)

**Comment by zendo** [comment]
Screenshot_20221128_200355

**Comment by zendo** [comment]
Weird, `nixpkgs-review` can't recognize `game-music-emu` alias name.

**Comment by bjornfor** [comment]
Result of `nixpkgs-review pr 200723` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>qmmp</li>
  </ul>
</details>

**Review by bjornfor — APPROVED** [review]

**Comment by bjornfor** [comment]
Thanks!

**Pull request merged**